packages/cozy-mespapiers-lib/src/utils/search.js

```diff
@@ -0,0 +1,120 @@
+import { Document } from 'flexsearch'
+
+import { models } from 'cozy-client'
+
+import { CONTACTS_DOCTYPE } from '../doctypes'
+
+const {
+  document: {
+    themes: { themesList }
+  },
+  file: { isFile }
+} = models
+
+export const index = new Document({
+  document: {
+    id: '_id',
+    tag: 'flexsearchProps:tag',
+    index: [
+      'flexsearchProps:name',
+      'flexsearchProps:translatedQualificationLabel',
+      'flexsearchProps:fullname',
+      'flexsearchProps:birthday',
+      'flexsearchProps:birthcity',
+      'flexsearchProps:email',
+      'flexsearchProps:civility',
+      'flexsearchProps:company',
+      'flexsearchProps:jobTitle'
```

Comment:
Why do you create this `flexsearchProps` object? This actually looks like a good idea, but to differenciate fields between doctype, eg. `io.cozy.files:name`, `io.cozy.contacts.fullname`, etc.

Reply:
parce qu'il faut créer des nouveaux attributs de toute façon (`tag`, ou `translatedQualificationLabel` par exemple), j'ai donc décidé de les regrouper dans un object bien distinct pour les isoler.